Makefile

```diff
@@ -1,6 +1,5 @@
 
 DOCKER ?= $(shell command -v docker)
-PANDOC ?= $(shell command -v pandoc)
```

Comment:
geez this is now a little messy. 
Perhaps this $(PANDOC) change best be reverted. Otherwise the whole command for the `docs` target will have to be overhauled to allow for a local pandoc binary.